workflows/prognostic_c48_run/runtime/names.py

```diff
@@ -19,13 +19,15 @@
 TSFC = "surface_temperature"
 MASK = "land_sea_mask"
 TIME_KEYS = ["time", "initialization_time"]
+X_WIND = "x_wind"
+Y_WIND = "y_wind"
+AGRID_U_TENDENCY = "dQu"
+AGRID_V_TENDENCY = "dQv"
```

Comment:
maybe use `EAST_WIND` and `NORTH_WIND` instead of `U` and `V`? Bit more consistent with usage elsewhere.